packages/terra-table/src/terra-dev-site/doc/example/TableSectionHeader.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+/* eslint-disable import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
+import Table, { SectionHeader } from 'terra-table/lib/index';
+/* eslint-enable import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
+
+const TableRowExample = () => (
+  <Table>
+    <SectionHeader title="Default SectionHeader" key="default" />
```

Comment:
lets add `colSpan` since it's a required prop

Reply:
Updated.